src/sql/workbench/contrib/notebook/browser/markdownWYSIWYGLinkHander.ts

```diff
@@ -0,0 +1,90 @@
+import { URI } from 'vs/base/common/uri';
+import * as path from 'vs/base/common/path';
+
+export class MarkdownWYSIWYGLinkHandler {
+	private _isAnchorLink: boolean | undefined;
+	private _isFile: boolean | undefined;
+	private _isAbsolutePath: boolean | undefined;
+	private _notebookDirectory: string;
+	private _notebookLink: URI | undefined;
+	private _href: string | undefined;
+
+	constructor(private _notebookURI: URI, private _pathOrElement: string | HTMLAnchorElement) {
+		if (typeof this._pathOrElement === 'string') {
+			this._notebookLink = URI.parse(this._pathOrElement);
+			this._isFile = this._notebookLink.scheme === 'file';
+			this._isAbsolutePath = path.isAbsolute(this._pathOrElement);
+			this._isAnchorLink = this._pathOrElement.includes('#') && this._isFile;
+		} else {
+			this._notebookLink = this._pathOrElement.attributes['href']?.nodeValue ? URI.parse(this._pathOrElement.attributes['href']?.nodeValue) : undefined;
```

Comment:
I was testing this branch against #15942 issue.
It seems that the absolute paths are working however it does not show the full absolute path (it should also include the scheme/drive it is stored in) ~ **C:\\Users** (unless this is intended)
![image](https://user-images.githubusercontent.com/23587151/125406891-2c0eb080-e36e-11eb-966a-3fcf1e096d84.png)

Also, this does not seem to work properly for other link schemes HTTPS for example _www.google.com_
![image](https://user-images.githubusercontent.com/23587151/125409321-a3ddda80-e370-11eb-8a37-d81f9fa8b853.png)
After inserting in WYSIWYG and changing to split view the link is not kept, and therefore does not work.